### DISCUSSION TITLE
Brainstorm da Feature Geração de Link de Compartilhamento

### DISCUSSION THREAD

**Discussion by crvzybird** [crvzybird/ess-cardapio#18] [Ideas]
## :link: Bem vindo(a)!
Essa **_Discussion_**  servirá como um espaço para realizar **_Brainstorms_** da **_Feature_**  **Geração de Link de Compartilhamento**  e qualquer etapa de desenvolvimento dela que esteja na sua fase de conceituação e ideação.Com o intuito de podermos nos manter conectados mesmo não sendo possível estar em reuniões virtuais/presenciais é esperado que comentários sejam feitos para debater possíveis abordagens, implementações e até sugestão de refatorações de implementações que possam contribuir no desenvolvimento dessa *_Feature_**

**Comment by mateusmeloxyz** [comment]
# Primeiro mockup

Baseando-se na interface do ifood, nosso app terá um link de compartilhamento no topo do cardapio/menu do estabelecimento.

![photo5030724009196628428](https://user-images.githubusercontent.com/52260710/159695617-e9cf4eb6-41d4-41e0-97d7-24a71ea301a3.jpg)
Posição do link de compartilhamento

![photo5030724009196628429](https://user-images.githubusercontent.com/52260710/159695842-454bc6f4-ac7a-421a-b758-647260aacfa6.jpg)
Icone do link de compartilhamento

O link exibirá um iframe sobreposto a aplicação que dará diversas opções de compartilhamento do cardápio/menu. Esta etapa será diferente no web app comparado ao app mobile nativo:

![photo5030724009196628430](https://user-images.githubusercontent.com/52260710/159696043-afa3895a-181c-4f2f-92f8-edde57a75095.jpg)
A versão web será similar ao compartilhamento do youtube, visto acima.

![photo5030724009196628431](https://user-images.githubusercontent.com/52260710/159696227-d9a9a4cb-8718-4827-8ae9-adbffd421903.jpg)
A versão mobile android usará o compartilhamento nativo dos dispositivos android.

**Reply by crvzybird** [reply]
@mateusgomes01
tu acha que seria interessante ter um compartilhamento via QR Code?

vi que tem uns sites que geram

aí a gente poderia ter um fixo mesmo

**Comment by mateusmeloxyz** [comment]
Sobre compartilhamento no facebook:

https://developers.facebook.com/docs/plugins/share-button/
